Jenkinsfile

```diff
@@ -107,10 +109,10 @@ pipeline {
       }
       steps {
         withGithubNotify(context: 'Intake') {
-          deleteDir()
-          unstash 'source'
-          dir("${BASE_DIR}"){
-            withGoEnv(){
+          runInPod(env.GO_VERSION) {
```

Comment:
you can interpolate variables in the pod definition, so you can launch small pods